gto/cli.py

```diff
@@ -619,6 +627,9 @@ def deprecate(
     simple: str = option_simple,
     force: bool = option_force,
     delete: bool = option_delete,
+    auto_push: bool = Option(
```

Comment:
A bug 🤔 
```shell
$ gto register bug --auto-push
Created git tag 'bug@v0.0.1' that registers version
Running `git push origin bug@v0.0.1`
Successfully pushed git tag bug@v0.0.1 to the remote.

$ gto deprecate bug --auto-push --delete
Deleted git tag 'bug@v0.0.1'
Running `git push origin bug@v0.0.1`
❌ Unexpected error: 'str' object has no attribute 'new_message_handler'
Please report it here running with '--traceback' flag: <https://github.com/iterative/gto/issues>
```

Reply:
argh! well spotted... My expectations for the test was wrong! I think I fixed it with this commit https://github.com/iterative/gto/pull/281/commits/ea2df8bfb63d7311b6d481cddb560fe23e7acbd3